src/signalhandler.cc

```diff
@@ -363,11 +392,15 @@ void InstallFailureSignalHandler() {
   for (size_t i = 0; i < ARRAYSIZE(kFailureSignals); ++i) {
     CHECK_ERR(sigaction(kFailureSignals[i].number, &sig_action, NULL));
   }
+#elif defined(OS_WINDOWS)
+  typedef void (*SignalHandlerPointer)(int); 
+  SignalHandlerPointer previousHandler;  
+  previousHandler = signal(SIGABRT, &FailureSignalHandler); 
```

Comment:
I _think_ this is because the Windows signal handler is not like what is found on Linux like systems. Instead, it uses structured exception handling (SEH). I don't know much more than what I've researched on this (ironically I am not a Windows dev, I am a Linux dev), but I know this was quite an annoying problem to solve. Signaling on Windows switches threads (to the SEH thread, I guess?) and `CaptureStackBackTrace` explicitly gets the backtrace for the current thread. I'm looking for a way to do this right.